homeassistant/components/synologydsm/sensor.py

```diff
@@ -150,10 +152,11 @@ def update(self):
 class SynoNasSensor(Entity):
     """Representation of a Synology NAS Sensor."""
 
-    def __init__(self, api, variable, variable_info, monitor_device=None):
+    def __init__(self, api, name, variable, variable_info,
+                 monitor_device=None):
         """Initialize the sensor."""
         self.var_id = variable
-        self.var_name = variable_info[0]
+        self.var_name = "{}_{}".format(name, variable_info[0])
```

Comment:
This is a friendly name, right? Why use `_` then? For friendly names we use spaces.